src/fromager/packagesettings.py

```diff
@@ -608,8 +608,15 @@ def get_extra_environ(
         # configure max jobs settings, settings depend on package, available
         # CPU cores, and available virtual memory.
         jobs = self.parallel_jobs()
+
+        templatemakeflags = template_env.get("MAKEFLAGS", "")
+        if templatemakeflags == "":
+            makeflags = f"-j{jobs}"
+        else:
+            makeflags = f"{templatemakeflags} -j{jobs}"
```

Comment:
If we just need to remove the blank spaces, we could use strip.

```suggestion
        makeflags = f"{templatemakeflags} -j{jobs}".strip()
```

Reply:
Oh that's a good idea.  I didn't know about strip().  I'll do that instead.

Reply:
Fixed in new push.